query/src/main/scala/filodb/query/exec/SetOperatorExec.scala

```diff
@@ -145,10 +145,10 @@ final case class SetOperatorExec(queryContext: QueryContext,
             lhs
           }
         } else lhs
-        val lhsRows = lhsStitched.rows
-        val rhsRows = rhsMap(jk).rows
+        val lhsRows = lhsStitched.rows()
```

Comment:
I think scalastyle will yell at you for these argument-less method calls-- how dare you indicate that this isn't just a field access?

Reply:
Hm-- evidently it will not

Reply:
All these are warnings in  IDEA